File-Formats/PDF/Assemblies-Required.md

```diff
@@ -6,7 +6,7 @@ control: PDF
 documentation: UG
 keywords: Assemblies
 ---
-# Assemblies Required to work with PDF 
+# Assemblies Required to work with .NET PDF 
```

Comment:
Use this - Assemblies Required to work with PDF in .NET

Reply:
Changed